desktop_version/src/FileSystemUtils.cpp

```diff
@@ -465,3 +469,49 @@ void PLATFORM_copyFile(const char *oldLocation, const char *newLocation)
 	/* WTF did we just do */
 	printf("Copied:\n\tOld: %s\n\tNew: %s\n", oldLocation, newLocation);
 }
+
+#ifdef _WIN32
+bool FILESYSTEM_openDirectoryEnabled()
+{
+	return true;
+}
+
+bool FILESYSTEM_openDirectory(const char *dname)
+{
+	ShellExecute(Null, "open", dname, NULL, NULL, SW_SHOWMINIMIZED);
```

Comment:
`NULL`, this is what tripped the test box!

Reply:
Whoops, that's slightly embarrassing. Fixed and force-pushed.

Reply:
If I had a dollar every time I swapped `null` and `NULL` when I was swapping between C and C#... >_>